src/m365/spo/commands/list/list-label-get.ts

```diff
@@ -79,32 +84,39 @@ class SpoListLabelGetCommand extends SpoCommand {
   }
 
   #initOptionSets(): void {
-    this.optionSets.push(['listId', 'listTitle']);
+    this.optionSets.push(['listId', 'listTitle', 'listUrl']);
   }
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
-    try {  
+    try {
       if (this.verbose) {
-        const list: string = (args.options.listId ? args.options.listId : args.options.listTitle) as string;
-        logger.logToStderr(`Getting label set on the list ${list} in site at ${args.options.webUrl}...`);
+        logger.logToStderr(`Getting label set on the list ${args.options.listId || args.options.listTitle || args.options.listUrl} in site at ${args.options.webUrl}...`);
       }
 
       let requestUrl: string = '';
 
       if (args.options.listId) {
         if (this.debug) {
-          logger.logToStderr(`Retrieving List Url from Id '${args.options.listId}'...`);
+          logger.logToStderr(`Retrieving List from Id '${args.options.listId}'...`);
         }
 
         requestUrl = `${args.options.webUrl}/_api/web/lists(guid'${formatting.encodeQueryParameter(args.options.listId)}')?$expand=RootFolder&$select=RootFolder`;
       }
-      else {
+      else if (args.options.listTitle) {
         if (this.debug) {
-          logger.logToStderr(`Retrieving List Url from Title '${args.options.listTitle}'...`);
+          logger.logToStderr(`Retrieving List from Title '${args.options.listTitle}'...`);
         }
 
         requestUrl = `${args.options.webUrl}/_api/web/lists/GetByTitle('${formatting.encodeQueryParameter(args.options.listTitle as string)}')?$expand=RootFolder&$select=RootFolder`;
       }
+      else if (args.options.listUrl) {
+        if (this.debug) {
+          logger.logToStderr(`Retrieving List from URL '${args.options.listUrl}'...`);
+        }
+
+        const listServerRelativeUrl: string = urlUtil.getServerRelativePath(args.options.webUrl, args.options.listUrl);
+        requestUrl = `${args.options.webUrl}/_api/web/GetList('${formatting.encodeQueryParameter(listServerRelativeUrl)}')`;
```

Comment:
here we could add the same small improvement to extract a similar part of the URL in the `requestUrl` variable